### ISSUE TITLE
Log Error: Invalid Argument

### ISSUE THREAD

**Issue by M47kus** [richibrics/IoTuring#1]
```
marku@Laptop-Markus MINGW64 ~
$ cd Downloads/

marku@Laptop-Markus MINGW64 ~/Downloads
$ git clone https://github.com/richibrics/IoTuring.git
Cloning into 'IoTuring'...
remote: Enumerating objects: 617, done.
remote: Counting objects: 100% (617/617), done.
remote: Compressing objects: 100% (371/371), done.
remote: Total 617 (delta 294), reused 497 (delta 184), pack-reused 0
Receiving objects: 100% (617/617), 119.18 KiB | 2.02 MiB/s, done.
Resolving deltas: 100% (294/294), done.

marku@Laptop-Markus MINGW64 ~/Downloads
$ cd IoTuring/

marku@Laptop-Markus MINGW64 ~/Downloads/IoTuring (main)
$ py main.py
[ 2022-08-29 11:41:39 |   Info   |              App              ]  Name: IoTuring
Traceback (most recent call last):
  File "C:\Users\marku\Downloads\IoTuring\main.py", line 23, in <module>
    logger.Log(Logger.LOG_INFO,"App",App()) # Print App info
  File "C:\Users\marku\Downloads\IoTuring\Logger\Logger.py", line 62, in Log
    self.Log(messageType, source, line)
  File "C:\Users\marku\Downloads\IoTuring\Logger\Logger.py", line 91, in Log
    self.PrintAndSave(string, messageType)
  File "C:\Users\marku\Downloads\IoTuring\Logger\Logger.py", line 131, in PrintAndSave
    self.GetLogFileDescriptor().write(string+' \n')
  File "C:\Users\marku\Downloads\IoTuring\Logger\Logger.py", line 135, in GetLogFileDescriptor
    self.log_file_descriptor = open(self.log_filename, "a")
OSError: [Errno 22] Invalid argument: 'C:\\Users\\marku\\Downloads\\IoTuring\\Logger\\Logs\\Log_2022-08-29_11:41:39.log'

marku@Laptop-Markus MINGW64 ~/Downloads/IoTuring (main)
$

```

**Comment by richibrics** [comment]
Hi, the log filename is not compatible with Windows. Give me a minute to solve this

**Comment by richibrics** [comment]
fixed in [3c4f157](https://github.com/richibrics/IoTuring/commit/3c4f157132dee1be7c2b23a4f97d5d79cc5490f0)

**Comment by richibrics** [comment]
Let me know if it works, thanks !
